### PR TITLE
fix: services section break points text breaking

### PR DESCRIPTION
**Pull request by aimee-annabelle**
The responsiveness of the services section was broken for some screens. This PR is for the fix on those breaks; all mobile, tablet, and big screens have been fixed.

### CODE REVIEW

**Review by josueahadi — APPROVED** [review]

**Review by gloriaumutoni — APPROVED** [review]
nice job

**Review by beylar — APPROVED** [review]

File: src/sections/services.html [hunk 1]
```diff
@@ -9,10 +9,15 @@
     <p>Digital marketing & content production</p>
     <p>Influencer relations</p>
   </div>
-  <div class="relative py-10 flex justify-end">
-    <img class="h-full z-0 flex self-end" src="./src/images/Vector_5.svg" />
+  <div
+    class="relative py-10 flex justify-end justify-self-end self-end sm:w-3/4 md:w-2/3 2xl:w-1/3"
+  >
+    <img
+      class="h-full xl:h-auto z-0 flex self-end"
```
Comment (Kingcedru) [inline src/sections/services.html:16]: Add image description

**Review by Kingcedru — CHANGES_REQUESTED** [review]
Some changes needed

File: src/sections/services.html (hunk 1 above)
Comment (aimee-annabelle) [reply src/sections/services.html:16]: resolved

**Review by Kingcedru — APPROVED** [review]

**Pull request merged**